legend-engine-xt-relationalStore-executionPlan-connection-tests/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/connection/test/ExternalIntegration_TestConnectionAcquisitionWithFlowProvider_Databricks.java

```diff
@@ -88,7 +86,7 @@ private RelationalDatabaseConnection databricksSpec() throws Exception
         dsSpecs.hostname = "dbc-f0687849-717f.cloud.databricks.com";
         dsSpecs.port = "443";
         dsSpecs.protocol = "https";
-        dsSpecs.httpPath = "sql/protocolv1/o/448477574000404/0228-234110-ve0xbrtk";
```

Comment:
Changing endpoint to SQL warehouse. Same environment, same creds, just a different endpoint